Assets/Scripts/PlayerMovementRecorder.cs

```diff
@@ -70,18 +71,23 @@ private void Save(int num)
         }
         else
         {
-            //File.Delete(path);
             //Write some text to the test.txt file
             writer = new StreamWriter(path, true);
         }
     }
 
     private void OnTriggerEnter(Collider other)
     {
+        BeeController beeController = other.GetComponent<BeeController>();
+        if (beeController != null)
+        {
+            writer.WriteLine("ReturnedToBee");
+        }
```

Comment:
Would be so good to have this code in a separate function, since the pattern is repeated all the time later, but I guess there's no need to change it now